client/src/ApiService/Meetings.ts

```diff
@@ -6,8 +6,20 @@ export const getMeetingDates = async () => {
     res.json(),
   )
   const meetingDates = response.map((meeting: Meeting) => {
-    return { meetingId: meeting.id, date: meeting.createdAt.slice(0, 10) }
+    return { meetingId: meeting.id, date: meeting.date.toString().slice(0, 10) }
   })
 
   return meetingDates
 }
+
+export const createMeetingInfo = async (meetingInfo: Meeting) => {
+  const response = await HttpRequest.post('/meetings', meetingInfo)
+  console.log(response)
+}
+
+export const getMeetingsInfoByUser = async (userId: number) => {
+  const response = await fetch(
+    HttpRequest.BASE_URL + `/meetings/${userId}`,
```

Comment:
별로 중요한 건 아니긴 한데 이렇게 쓰시면 더 간단할 것 같아서 코멘트 남겨요 :)

```suggestion
}

export const getMeetingsInfoByUser = async (userId: number) => {
  const response = await fetch(
   `${HttpRequest.BASE_URL}/meetings/${userId}`,
```

Reply:
아, 그리고 REST API 네이밍할 때 `/meetings/${id}`와 같이 쓰는 경우는 여기서 `id`가 `meetingId`인 경우에 이렇게 써요. 이 경우에는 `meetingId`가 아니라 `userId`기 때문에 query param을 사용하셔서 `/meetings?user_id=${userId}` 이런 식으로 쓰시는 게 더 적합해요.